resources/definitions/fdmprinter.def.json

```diff
@@ -4424,6 +4424,19 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "prime_tower_purge_volume":
+                {
+                    "label": "Prime Tower Purge Volume",
+                    "description": "Amount of filament to be purged when wiping on the prime tower. Purging is useful for compensating the filament lost by oozing during inactivity of the nozzle.",
+                    "type": "float",
+                    "enabled": "dual_pre_wipe",
```

Comment:
> The setting should be enabled when the prime tower is enabled and the wipe is enabled.

LipuFei 2 hours ago Member
> wipe already depends on the prime tower so that's not necessary